packages/ee/billing/handlers/subscription-deleted.ts

```diff
@@ -20,45 +12,18 @@ export const handleSubscriptionDeleted = async (event: Stripe.Event) => {
   }
 
   const organization = await getOrganization(organizationId);
-  if (!organization) throw new Error("Organization not found.");
-
-  const updatedFeatures = organization.billing.features;
-
-  for (const item of stripeSubscriptionObject.items.data) {
-    const product = await stripe.products.retrieve(item.price.product as string);
-
-    switch (product.name) {
-      case StripeProductNames.inAppSurvey:
-        updatedFeatures[ProductFeatureKeys.inAppSurvey as keyof typeof organization.billing.features].status =
-          "inactive";
-        updatedFeatures[
-          ProductFeatureKeys.inAppSurvey as keyof typeof organization.billing.features
-        ].unlimited = false;
-        await unsubscribeCoreAndAppSurveyFeatures(organizationId);
-        break;
-      case StripeProductNames.linkSurvey:
-        updatedFeatures[ProductFeatureKeys.linkSurvey as keyof typeof organization.billing.features].status =
-          "inactive";
-        updatedFeatures[
-          ProductFeatureKeys.linkSurvey as keyof typeof organization.billing.features
-        ].unlimited = false;
-        await unsubscribeLinkSurveyProFeatures(organizationId);
-        break;
-      case StripeProductNames.userTargeting:
-        updatedFeatures[
-          ProductFeatureKeys.userTargeting as keyof typeof organization.billing.features
-        ].status = "inactive";
-        updatedFeatures[
-          ProductFeatureKeys.userTargeting as keyof typeof organization.billing.features
-        ].unlimited = false;
-        break;
-    }
-  }
+  if (!organization) throw new ResourceNotFoundError("Organization not found", organizationId);
 
   await updateOrganization(organizationId, {
     billing: {
       ...organization.billing,
-      features: updatedFeatures,
+      plan: PRODUCT_FEATURE_KEYS.FREE,
+      limits: {
+        monthly: {
+          responses: LIMITS.FREE.RESPONSES,
```

Comment:
We sometimes set them hardcoded (e.g. in the prisma schema default value) and sometimes with these constant. Should we maybe don't set them on the database level using prismas default value and instead also use these constants when creating a new organization in the organization service. Then we can also only set them if IS_FORMBRICKS_CLOUD